lib/active_shipping/location.rb

```diff
@@ -160,6 +160,7 @@ def address2_and_3
     end
 
     def ==(other)
+      return false if !other.respond_to?(:to_hash)
```

Comment:
I'd go for something like
```patch
     def ==(other)
-      to_hash == other.to_hash
+      to_hash == other&.to_hash
     end
```

Reply:
That’s a newer ruby feature.. is that ok to do?

Reply:
I think `&.` was introduced in Ruby 2.3 and Travis is still testing 2.2.

Reply:
Not sure about that! Ruby 2.3 was introduced 2 years ago, but probably Thomas has an answer :)

Reply:
Yea, we wouldn't be able to use the safe navigator operator here, since `ActiveShipping` is still supporting 2.2.